src/js/node/fs.ts

```diff
@@ -599,10 +600,13 @@ var access = function access(path, mode, callback) {
     return new FSWatcher(path, options, listener);
   },
   opendir = function opendir(path, options, callback) {
+    // TODO: validatePath
+    // validateString(path, "path");
     if (typeof options === "function") {
       callback = options;
       options = undefined;
     }
+    validateFunction(callback, "callback");
     const result = new Dir(1, path, options);
     if (callback) callback(null, result);
```

Comment:
`if (callback)` can be removed since it is validated to be a function

Reply:
fixed